ros2topic/test/test_cli.py

```diff
@@ -559,7 +559,14 @@ def test_topic_echo_no_publisher(self):
 
     def test_topic_pub(self):
         with self.launch_topic_command(
-            arguments=['pub', '/chit_chatter', 'std_msgs/msg/String', '{data: foo}'],
+            arguments=[
+                'pub',
+                '--keep-alive', '3',
```

Comment:
@dirk-thomas nit: 
```suggestion
                '--keep-alive', '3',  # seconds
```
Same below.

Reply:
Done in f5cfb27de2bc75d4612d17418bac5ecaae3467f0.